src/_base.scss

```diff
@@ -1,6 +1,8 @@
+@use "./colors" as *;
```

Comment:
What does `as *` do? Currently you're just making it work, without using the full advantage. Read the docs carefully about what it is used for and strictly follow the docs.

Reply:
Just read `@use "./colors" as *`. It is a plain english. You can tell what `as` keyword does right away.